tools/fix_auth.rb

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
-require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+require File.expand_path('../config/environment', __dir__)
```

Comment:
this is the simpler approach, but I'm pretty sure we don't want to include this by default